tiny-postgres-event-store-spring-boot-starter/src/main/resources/migrations/V1__init.sql

```diff
@@ -0,0 +1,36 @@
+create schema if not exists event_sourcing_store;
+create sequence event_sourcing_store.event_record_id_sequence;
+create table if not exists event_sourcing_store.event_record (
+       id text default nextval('event_sourcing_store.event_record_id_sequence')::text,
```

Comment:
it breaks current implementation - in code you can see that EventRecord has compound key - 
```            EventRecord(
                "$aggregateId-${it.version}",
                aggregateId,
                it.version,
                it.name,
                eventMapper.eventToString(it),
                it.sagaContext
            )```

making it generating from sequence you break the optimistic concurrency which is based on clashing the id of the event records.

Reply:
better use sequence for createAt field